plugins/foreman_virt_who_configure/0.1/index.md

```diff
@@ -0,0 +1,173 @@
+ 
+There's also a widget dasboard that you can watch. It contains a table with number of configurations per each state. If you click on status name, you'll see all configurations in this state. The widget also lists last 3 configuration in __No Change__ state as this might or might not be problematic.
+ 
+On configurations lists page you can also search based on the state. The search syntax can be used
+
+```
+status = unknown
+status = out_of_date
+status = ok
+status = ok or status = out_of_date
+```
+
+## 4.4 Redeploying
+
+If there's some change you need to do to existing virt-who configuration, simply edit it via the same form that was used during creation. To edit a configuration, click _Edit_ button in configuration table on a particular row. After you submit new values, the configuration script reflects it. You can simply run the new version of the script and it will reconfigure the virt-who according to new values. It also restart the service for you. You can run the script repeatedly.   
+
+## 4.5 Authentication and authorization
+
+Plugin creates few roles that can be useful even without using the configuration functionality.
+
+* **Virt-who Manager** - this role contains all permission to manage virt-who configurations, user needs this role to create/delete/update configurations.
+* **Virt-who Reporter** - this is a role that contains minimal set of permissions for virt-who to upload the report, it can be used if you configure virt-who manually and want to use user that has locked down account
+* **Virt-who Viewer** - this user can see all configurations including their configuration scripts, which means viewers could still deploy the virt-who instances for existing virt-who configurations. 
+
+Every virt-who configuration create a service user which is used for virt-who instance authentication. This means every virt-who configuration uses a different account. The service user login is _virt_who_reporter_$id_ and has randomly generated password. The user account is assigned _Virt-who Reporter_ role. The service user can't login to Foreman and can only be used for virt-who reporting. 
+
+# 5. API
+
+All functionality that is provided by WebUI can be also achieved by REST API. To find information about the available API endpoints, visit `your.foreman.com/apidoc`. In addition to common 7 routes for managing virt-who configurations, we have one endpoint for generating the configuration script. 
+
+# 6. Hammer
+
+The Hammer plugin is available too. To install it, install the Hammer plugin package from katello repostories. Note for Satellite 6 users, you don't need to take any action, as long as you have satellite-cli package installed, the plugin is configured automatically. 
```

Comment:
This applies only to Satellite 6, I'd like to avoid confusion and was hoping that we can mention the difference here. I don't think there's similar package upstream.